bpf/lib/lxc.h

```diff
@@ -65,6 +65,35 @@ static inline int is_valid_lxc_src_ipv4(struct iphdr *ip4)
 }
 #endif
 
+/**
+ * skb_redirect_to_proxy configures the skb with the proxy mark and proxy port
+ * number to ensure that the stack redirects the packet into the proxy.
+ *
+ * It is called from both ingress and egress side of endpoint devices.
+ *
+ * In regular veth mode:
+ * * To apply egress policy, the egressing endpoint configures the mark here,
+ *   which returns TC_ACT_OK to pass the packet to the stack in the context
+ *   of the source device (stack ingress).
+ * * To apply ingress policy, the egressing endpoint or netdev program tail
+ *   calls into the policy program which configures the mark here, which
+ *   returns TC_ACT_OK to pass the packet to the stack in the context of the
+ *   source device (netdev or egress endpoint device, stack ingress).
+ *
+ * In chaining mode with bridged endpoint devices:
+ * * To apply egress policy, the egressing endpoint configures the mark here,
```

Comment:
a nit...
  "the egressing endpoint configures the mark here,"

I assume "here" is this function? So sounds better to drop the 'here' IMO,

 "the egressing endpoint configures the mark,"